sonic-xcvrd/xcvrd/sff_mgr.py

```diff
@@ -435,6 +435,12 @@ def task_worker(self):
                     # Skip if these essential routines are not available
                     continue
 
+                try:
+                    api = sfp.get_xcvr_api()
+                    api.set_lpmode(False)
```

Comment:
@peterbailey-arista Per the below implementation, an exception will not be returned for SFF-8472 modules. Can you please handle this accordingly.
https://github.com/sonic-net/sonic-platform-common/blob/0f2e22faccd093a1e5d18235fe119a860be7855e/sonic_platform_base/sonic_xcvr/api/public/sff8472.py#L308

Reply:
I've now updated it to use sfp.set_lpmode() for only SFPs implementing SFF8472 as was originally suggested